pkg/lca/imagebasedupgrade.go

```diff
@@ -223,6 +223,23 @@ func (builder *ImageBasedUpgradeBuilder) WithSeedImage(
 	return builder
 }
 
+// WithOadpContent adds oadpContent to be used by the imagebasedupgrade.
+// This is used for backup/restore during upgrade.
+func (builder *ImageBasedUpgradeBuilder) WithOadpContent(
+	oadpContentConfigMapName string, oadpContentConfigMapNamespace string) *ImageBasedUpgradeBuilder {
```

Comment:
No need to repeat argument type sting.
You can do just >
oadpContentConfigMapName, oadpContentConfigMapNamespace string

Reply:
updated. thanks.